lmfdb/groups/abstract/main.py

```diff
@@ -1261,15 +1261,17 @@ def group_parse(info, query):
         CheckCol("solvable", "group.solvable", "solv", short_title="Sub. solvable"),
         CheckCol("maximal", "group.maximal_subgroup", "max", short_title="Sub. maximal"),
         CheckCol("perfect", "group.perfect", "perf", short_title="Sub. perfect"),
-        CheckCol("central", "group.central", "cent", short_title="Sub. central")]),
+        CheckCol("central", "group.central", "cent", short_title="Sub. central"),
+        ProcessedCol("sylow", "group.sylow_subgroup", "Sylow", lambda x: f"${latex(x)}$" if x > 1 else "", align="center", short_title="Sub. Sylow"),
```

Comment:
Please move this line (which gives us the Sylow column) up a few rows so the Sylow column shows up immediately after the Order column.  Everything else looks great.